### PR TITLE
Add thoth config

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Add thoth config to support version releases

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+runtime_environments:
+  - name: 'rhel:8'
+    operating_system:
+      name: rhel
+      version: "8.0"
+    python_version: "3.6"
+    recommendation_type: latest
+    limit_latest_versions: 5
+
```
Comment (harshad16) [inline .thoth.yaml:13]: we dont need this here as well, right?

File: version.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+"""Thoth Application."""
+
+__name__ = "kebechet"
```
Comment (harshad16) [inline version.py:3]:
```suggestion
__name__ = "thoth-station"
```
or something else?

File: .thoth.yaml (hunk 1 above)
Comment (saisankargochhayat) [reply .thoth.yaml]: We can remove these

**Review by harshad16 — APPROVED** [review]
Thanks :100:

File: .thoth.yaml (hunk 1 above)
Comment (fridex) [reply .thoth.yaml]: Yes, `limit_latest_versions: 5` is no longer supported. If you spot it used somewhere, please remove it from `.thoth.yaml`.

**Pull request merged**